src/include/ipc/header.h

```diff
@@ -19,18 +19,91 @@
 #include <stdint.h>
 
 /** \addtogroup sof_uapi uAPI
- *  SOF uAPI specification.
+ * SOF uAPI specification
+ * =======================
  *
- * IPC messages have a prefixed 32 bit identifier made up as follows :-
+ * Overview
+ * --------
+ *
+ * The SOF Audio DSP firmware defines an Inter-Process Communication
+ * (IPC) interface to facilitate communication with the host.
+ *
+ * The SOF IPC is bi-directional. Messages can be initiated by the
+ * host and acknowledged by the DSP. Similarly, they can be initiated
+ * by the DSP and acknowledged by the host.
+ *
+ * IPC messages are divided into several groups: global, topology,
+ * power management, component, stream, DAI, trace, and a separate
+ * "firmware ready" message. Multiple messages can also be grouped
+ * into a message that belong to a compound group. Most messages are
+ * sent by the host to the DSP; only the following messages are sent
+ * by the DSP to the host:
+ *  - firmware ready: sent only once during initialization
+ *  - trace: optional, contains firmware trace data
+ *  - position update: only used if position data cannot be
+ *    transferred in a memory window or if forced by the kernel
+ *    configuration
+ *
+ * Message structure
+ * -----------------
+ *
+ * IPC messages have a fixed header and variable length payload.
+ *
+ *     +-------------------------------+
+ *     | IPC_HEADER (sof_ipc_cmd_hdr)  |
+ *     +-------------------------------+
+ *     | IPC_PAYLOAD (type specific)   |
+ *     +-------------------------------+
+ *
```

Comment:
Can you make the `IPC_PAYLOAD` box two lines longer? Just to make it look visually different from the header. Same below.